server/app/services/barman-service.js

```diff
@@ -90,10 +91,88 @@ async function deleteBarmanById(barmanId) {
     return barman;
 }
 
+/**
+ * Get the services of a barman
+ *
+ * @param barmanId {number} barman id
+ * @returns {Promise<Array>} Barmen's services
+ */
+async function getBarmanServices(barmanId) {
+
+    logger.verbose('Barman service: retreive services of the barman ', barmanId );
+
+    const barman = await Barman.findById(barmanId);
+
+    if (!barman) throw createUserError('UnknownBarman', 'This Barman does not exist');
+
+    return await Service.findAll({
```

Comment:
Because sequelize is really good, you can simply do a `return await barman.getServices()` (to test)